src/commands/protocols/common/protocol-message-command.js

```diff
@@ -12,29 +8,11 @@ class ProtocolMessageCommand extends Command {
     }
 
     async executeProtocolMessageCommand(command, messageType) {
-        if (!(await this.shouldSendMessage(command))) {
-            return Command.empty();
-        }
         const message = await this.prepareMessage(command);
 
         return this.sendProtocolMessage(command, message, messageType);
     }
 
-    async shouldSendMessage(command) {
```

Comment:
Shouldn't we do this only for publish and not for get. We don't need to send messages if get is finished?